### PR TITLE
[1LP][RFR] block test with BZ-1729594

### PR DESCRIPTION
**Pull request by digitronik**
Signed-off-by: Nikhil Dhandre <ndhandre@redhat.com>

{{pytest: cfme/tests/automate/custom_button/test_container_objects.py::test_custom_button_dialog_container_obj --use-provider ocp-39-hawk  -v}}

### CODE REVIEW

File: cfme/tests/automate/custom_button/test_service_vm_custom_button.py [hunk 1]
```diff
@@ -136,7 +137,8 @@ def test_custom_button_display_service_vm(request, appliance, service_vm, button
 
 @test_requirements.customer_stories
 @pytest.mark.tier(1)
-@pytest.mark.meta(automates=[1687061])
+# Dynamic dialog problem 1729594
+@pytest.mark.meta(automates=[1687061], blockers=[BZ(1729594), BZ(1729594)])
```
Comment (sshveta) [inline cfme/tests/automate/custom_button/test_service_vm_custom_button.py]: Why do we have same BZ twice ?

Comment (mshriver) [reply cfme/tests/automate/custom_button/test_service_vm_custom_button.py]: One blocks tests, one tracks automation for BZ test coverage.

When the test is no longer blocked by this BZ, blockers can be removed, automates continues to track that this test function/module/etc covers that BZ.

Comment (sshveta) [reply cfme/tests/automate/custom_button/test_service_vm_custom_button.py]: @mshriver : Shouldn't it be this then ?

     @pytest.mark.meta(automates=[1687061, 1729594], blockers=[BZ(1729594)])

Comment (digitronik) [reply cfme/tests/automate/custom_button/test_service_vm_custom_button.py]: @sshveta its was my bad its should `BZ(1729046), BZ(1729594)` :+1: 

about `automate` or `coverage` marker. It should not cover all BZ only `steps` not `setup` or `teardown`

about blocker; It should block as per version (5.10 and 5.11 but i missed; you can take it as `Human error`). Other thing about blocker, As reviewer my self feel bad about this; when, I asked someone to remove `force_stream` like thing as its depends on release version which depends on `triage`. contributors added this after adding BZ and we asked to remove it as per current status.

**Pull request merged**